Mathlib/MeasureTheory/Integral/Periodic.lean

```diff
@@ -221,14 +225,56 @@ protected theorem intervalIntegral_preimage (t : ℝ) (f : UnitAddCircle → E)
 
 end UnitAddCircle
 
-variable {E : Type*} [NormedAddCommGroup E] [NormedSpace ℝ E]
-
+/-!
+## Interval integrability of periodic functions
+-/
 namespace Function
 
 namespace Periodic
 
+variable {E : Type*} [NormedAddCommGroup E]
+
 variable {f : ℝ → E} {T : ℝ}
 
+/-- A periodic function is interval integrable over every interval if it is interval integrable
+over one period. -/
+theorem intervalIntegrable {t : ℝ} (h₁f : Function.Periodic f T) (hT : 0 < T)
+  (h₂f : IntervalIntegrable f MeasureTheory.volume t (t + T)) (a₁ a₂ : ℝ) :
+    IntervalIntegrable f MeasureTheory.volume a₁ a₂ := by
+  -- Replace [a₁, a₂] by [t - n₁ * T, t + n₂ * T], where n₁ and n₂ are integers
+  obtain ⟨n₁, hn₁⟩ := exists_nat_ge ((t -min a₁ a₂) / T)
+  obtain ⟨n₂, hn₂⟩ := exists_nat_ge ((max a₁ a₂ - t) / T)
+  have : Set.uIcc a₁ a₂ ⊆ Set.uIcc (t - n₁ * T) (t + n₂ * T) := by
+    rw [Set.uIcc_subset_uIcc_iff_le]
+    constructor
+    · calc min (t - ↑n₁ * T) (t + ↑n₂ * T)
```

Comment:
Do you need the arrows ↑? I'd tend to omit them, if they don't change the statement.

Reply:
Same below.

Reply:
Removed arrows.